### PR TITLE
add loading screen

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

**Comment by varffs** [comment]
@mikeyrayvon is the idea to show loading until the whole scene is done or just the first object is loaded?

**Comment by mikeyrayvon** [comment]
@varffs until the whole scene is done.  i only have one model in my scene,  i guess that progress function runs for every model.  can you fix?

**Comment by varffs** [comment]
Sure will do

**Comment by varffs** [comment]
Passing this to you @mixcas cuz I made significant changes and you need to check.

File: img/icons/logo.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg class="logo" xmlns="http://www.w3.org/2000/svg" width="2407.214" height="888" viewBox="0 0 2407.214 888"><path d="M552.756 144.117l123.723 123.73v-123.73zM325.553 744.474h129.965L325.553 614.516zM325.553 544.116l200.36 200.36h133.708L325.554 410.44zM325.553 340.02L676.48 690.947v-143.64L325.552 196.39zM1125.437 144.117l-123.743 123.73v-123.73zM1352.627 744.474h-129.97l129.97-129.958zM1352.627 544.116l-200.348 200.36h-133.71l334.057-334.038zM1352.627 340.02l-350.933 350.928v-143.64l350.933-350.917zM1963.33 144.117l-123.718 123.73v-123.73zM2190.532 744.474h-129.965l129.965-129.958zM2190.532 544.116l-200.36 200.36H1856.47l334.062-334.038zM2190.532 340.02l-350.92 350.928v-143.64l350.92-350.917zM1545.77 143.485l-123.737 123.73v-123.73zM1772.972 743.84h-129.97l129.97-129.95zM1772.972 543.48l-200.354 200.365H1438.91l334.062-334.04zM1772.972 339.4l-350.94 350.93V546.674l350.94-350.917zM0 779.93h2407.214V888H0zM0 0h2407.214v108.692H0zM343.676 144.117h138.678l194.132 194.135v138.67zM1334.498 144.117h-138.66l-194.15 194.135v138.67zM2172.41 144.117h-138.68L1839.62 338.252v138.67zM1754.843 143.485h-138.678L1422.04 337.617v138.665z"/></svg>
```
Comment (mixcas) [inline img/icons/logo.svg]: @mikeyrayvon how is this file different from the on we use in the nav (top-right)? is this duplicate? can we reuse the other one?

**Pull request merged**